lookout/style/format/analyzer.py

```diff
@@ -35,24 +41,61 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
     @with_uasts_and_contents
     def train(cls, ptr: ReferencePointer, config: dict, data_request_stub: DataStub,
               **data) -> AnalyzerModel:
+        """
+        Train a model given the files available.
+
+        :param url: url of the repository
+        :param commit: hash of the commit that we analyze
+        :param config: configuration dict
+        :param data: data sent by the lookout server
+        :return: a modelforge.Model containing the learned rules, per language.
+        """
         cls.log.info("train %s %s %s", ptr.url, ptr.commit, data)
         files = data["files"]
+        language_to_files = defaultdict(list)
         for file in files:
+            if not len(file.uast.children):
+                continue
+            # TODO: replace this temporary hack when langauge fields in File objects are fixed
```

Comment:
I have added the TODO format: https://github.com/src-d/guide/blob/d76e64ae32a9afbdfcf8fbefdecf683459176d79/engineering/conventions-python.md#code-style

Copied from Google, this allows asking the person who wrote the TODO what he meant without running `git blame`